contracts/src/v0.8/Owned.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title The Owned contract
+ * @notice A contract with helpers for basic contract ownership.
+ */
+contract Owned {
```

Comment:
`Owned` already exists in `v0.8/dev` but it has been renamed to `ConfirmedOwner`. Let's delete this one.